### DISCUSSION TITLE
Cicadas identified as Barn Owls

### DISCUSSION THREAD

**Discussion by dalogue1** [Nachtzuster/BirdNET-Pi#163] [General]
I know there is nothing I can really do about it but Cicadas (common in Texas) are misidentified as Barn Owls.

Luckily, they are on 17-year and 13-year cycles so they will be gone after this year for a while!

**Comment by lloydbayley** [comment]
That's really weird. I have cicadas here in Australia that are sometimes deafening but they have never been detected by my BirdNet. I'll have to keep an eye on it when it gets to summer here. (Assuming they decide to come out this year...)

**Reply by dalogue1** [reply]
What is the probability of Barn Owls in your location? 

Listen to how they sound. i can understand how they are confused with a Cicada.

https://www.allaboutbirds.org/guide/Barn_Owl/sounds#

**Reply by lloydbayley** [reply]
Interesting...gave that a listen. The Eastern NSW (is my region) recording could certainly be mistaken for short cicada burst. The cicadas here usually all start up together and go for aaaaaaggggessss and then they all fade away at the same time. It's very noisy for long periods, nothing like the short bursts so I suppose that's why I don't get the mis-indentifications but I will keep an eye on it! Here's an example of the [noise](https://www.youtube.com/watch?v=TaEVXsG74Bk). That video is close to one to get it's individual noises but all the background noise is more like our usual.

**Comment by wenestvedt** [comment]
I'll trade you: at one in the morning this week, I got an "identification" of a Common Loon. But I suspect it was an owl!

I know loons from growing up in Minnesota; here in Rhode Island, I haven't seen (or heard) one in thirty+ years.

**Reply by lloydbayley** [reply]
Loons are always a problem....and not always birds!! :)

**Reply by dalogue1** [reply]
Well it looks like at least there will be a future feature to prevent reporting these "detections" to BirdWeather.  https://github.com/Nachtzuster/BirdNET-Pi/issues/161

**Comment by PaulRB** [comment]
Our Silkie cockerel is frequently identified as a Greylag Goose, and occasionally other species!
